miio/airconditioningcompanion.py

```diff
@@ -208,7 +227,7 @@ def fan_speed(self) -> Optional[FanSpeed]:
     def mode(self) -> Optional[OperationMode]:
         """Current operation mode."""
         try:
-            mode = int(self.data[1][3:4])
+            mode = int(self.data['model_and_state'][1][3:4])
```

Comment:
Considering all of these access the same structure, would it make sense to have a variable for it? Like `self.state = self.data['model_and_state'][1]`?

Reply:
Good point!